src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 3 && days < 7) {
+    return (days * 40) - 20;
```

Comment:
move 40, 20, 50 into constant out of function, keep in mind that constants should be uppercased